Makefile

```diff
@@ -125,3 +124,10 @@ phpstan: vendor $(PHPSTAN)
 
 validate:
 	composer validate --strict
+
+bin/infection.phar: app bin src vendor box.json.dist scoper.inc.php box.phar
```

Comment:
There was a `build` folder created for generated PHAR since https://github.com/infection/infection/pull/67

IMO it would be consistent to have all non-commited files in one folder to not pollute the source code, what do you think?

Reply:
I'm fine with that